### DISCUSSION TITLE
Can't create PR because they "are entirely different commit histories"

### DISCUSSION THREAD

**Discussion by parsley72** [community/community#78903] [Pull Requests]
### Select Topic Area

Question

### Body

I forked https://github.com/jbulow/tinyxpath and did some work on it in https://github.com/parsley72/tinyxpath. Github shows the Contribute button which leads to "Open pull request" but when I click it I get:
> There isn’t anything to compare.
> jbulow:main and parsley72:main are entirely different commit histories.

How is this possible when it's a fork? Is there anyway of overriding this?

**Comment by jsoref** [comment]
So, the network for these two repositories looks *very* confusing: https://github.com/parsley72/tinyxpath/network

There's a commit: https://github.com/parsley72/tinyxpath/commit/f76190d691c7eb13873e94016ecb40d1037ce3c8 which says it's `Initial commit`, but it has a parent: https://github.com/parsley72/tinyxpath/commit/7f340f217fa196ffd745c4f6e1271116d4b1d66c -- and that doesn't appear to be you...

Notably, that commit appears to relate to: https://github.com/parsley72/tinyxpath/releases/tag/Version_1_3_1

I can't figure out what happened. The easiest way to get to a state like this is if you use `git rebase` of someone else's history onto another lineage.

While it is _possible_ to harmonize these two things, if I were to receive a PR that included such harmonization, I'd reject it.

So... The question is then: what are you actually trying to do, and why?

The easiest thing to do would be to recreate your changes against the upstream's commits and be careful not to mutate them.
